### ISSUE TITLE
How to use org-babel integration without SSH key authentication

### ISSUE THREAD

**Issue by MicLennartz** [vincentbernat/junos-mode#4]
Hi,
as far as I know, your module is relying on the fact, that SSH key authentication is available.
Is there an easy way to add support for some SSH password authentication, e.g as fallback in case the key authentication fails ?

Or do you have any other recommendation to address such cases ? (I've tried SSH session multiplexing as workaround, but doesn't seem to work with PyEz)

Br,
Michael

**Comment by vincentbernat** [comment]
Did you try `login:password@hostname`?

**Comment by MicLennartz** [comment]
Yes, that's working. But I can't save such files with plain-text passwords to disk then.

**Comment by vincentbernat** [comment]
What could be the alternative? A configuration file? Prompting for the password?

**Comment by MicLennartz** [comment]
I guess, another optional babel header argument like ":password" would be the easiest.
If that's set, the ob-junos.el could do a cached password prompt:

(set 'pass (password-read "Password? " "net-login"))
(password-cache-add "net-login" 'pass)

... and use that for the connection.
Unfortunately my understanding on elisp and org/babel is currently very limited and I didn't get such approach working.

(This is still not very secure, but at least nobody can easily fetch my password from the screen or filesystem)

**Comment by vincentbernat** [comment]
Yes, this would be a sensible alternative. I don't know when I will have time to do that.
